limitador-server/src/http_api/server.rs

```diff
@@ -32,6 +41,10 @@ impl RateLimitData {
     fn metrics(&self) -> &PrometheusMetrics {
         self.metrics.as_ref()
     }
+
+    fn status(&self) -> Status {
```

Comment:
Do you want to deref?
You could return &Status, like the others, and faster?

Reply:
No you can't cause that's guarded by a `RwLock`. The reference can't outlive the `RwLockReadGuard`.
So instead of reading the fields, which are just a few bytes, and return that, I'd opted for "snapshot'ing" the state in a `Copy`.